_node-machine_to_Syncano-script/test.js

```diff
@@ -0,0 +1,54 @@
+const axios = require('axios');
+const bluebird = require('bluebird');
+const machines = require('./data.json');
+const functionsArray = [];
+const BLACK_LIST = [
+  'machinepack-elasticsearch',
+  'machinepack-assets',
+  'machinepack-weather',
+  'machinepack-azure',
+  'machinepack-mongodb',
+  'machinepack-thumb',
+];
+
+const setCallsArray = () => {
+  const installSocket = (name, install_url) => {
```

Comment:
there were 2 functions inside, I refactored it, and forgot to delete the outer function. thanks